### DISCUSSION TITLE
How was Rocket's website created?

### DISCUSSION THREAD

**Discussion by brundonsmith** [rwf2/Rocket#2108] [General]
Hi! I was just thinking, every time I see Rocket's website I admire it (for its design, UX, etc)

I see the data under the site/ directory, but I was wondering whether the actual implementation is all custom, or whether you used a static site generator? (or Rocket itself?) Also, did you hire a designer or do the styling from scratch (or use an off-the-shelf template)? Were the drawings/icons custom or did you find them somewhere?

Just curious! Cheers

**Comment by nCrazed** [comment]
https://github.com/SergioBenitez/Rocket/tree/master/site
